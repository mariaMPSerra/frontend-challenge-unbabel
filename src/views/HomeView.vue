<template>
  <div class="home">
    <AppHeader />
    <Transcriptions />
  </div>
</template>

<script setup lang="ts">
import Transcriptions from '../components/home/AppTranscriptions.vue'
import AppHeader from '../components/home/AppHeader.vue'
</script>

<style lang="scss">
.home {
  box-shadow: 0px 3px 5px $gray;
  margin: 10px;
  border: 1px solid $border-color;
}
</style>
