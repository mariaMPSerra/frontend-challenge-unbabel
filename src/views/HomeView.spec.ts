import { describe, expect, it } from 'vitest'
import HomeView from './HomeView.vue'
import { shallowMount } from '@vue/test-utils'

describe('AppHome Component', () => {
  it('renders AppHome component', async () => {
    const wrapper = shallowMount(HomeView)
    const AppHeader = wrapper.findComponent({ name: 'AppHeader' })
    const Transcriptions = wrapper.findComponent({ name: 'Transcriptions' })

    expect(wrapper.html()).toMatchSnapshot()
    expect(AppHeader.exists()).toBe(true)
    expect(Transcriptions.exists()).toBe(true)
  })
})
