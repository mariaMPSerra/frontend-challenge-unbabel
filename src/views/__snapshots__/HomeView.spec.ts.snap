// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AppHome Component > renders AppHome component 1`] = `
"<div class="home">
  <app-header-stub></app-header-stub>
  <transcriptions-stub></transcriptions-stub>
</div>"
`;
